.github/dependabot.yml

```
@@ -8,3 +8,12 @@ updates:
       actions:
         patterns:
           - "*"
+
+  - package-ecosystem: pip
+    directory: "/"
+    schedule:
+      interval: monthly
```

```suggestion
    schedule:
      interval: monthly
    assignees:
      - "ezio-melotti"
```
If would like me to review (and possibly merge) these PRs, I don't mind have them auto-assigned to me.